Lib/pathlib/_abc.py

```diff
@@ -11,6 +11,7 @@
 resemble pathlib's PurePath and Path respectively.
 """
 
+import errno
```

Comment:
It seems we import explicit constants from `stat`. Maybe import `EINVAL` from `errno` as well (that's the only one being used I think).